superset-frontend/src/SqlLab/components/AceEditorWrapper/useKeywords.ts

```diff
@@ -55,10 +55,23 @@ const getHelperText = (value: string) =>
     docText: value,
   };
 
+const extensionsRegistry = getExtensionsRegistry();
+
+const useDefaultCustomKeywords = () => ({ data: undefined });
+
 export function useKeywords(
   { queryEditorId, dbId, schema }: Params,
   skip = false,
 ) {
+  const useCustomKeywords =
```

Comment:
Can we instantiate customKeywords as an empty array that is overridden when  `sqleditor.extension.customAutocomplete` is defined?